src/pydvl/utils/parallel/check.py

```diff
@@ -0,0 +1,47 @@
+import inspect
+from functools import partial
+from typing import Callable
+
+
+def check_fn_accepts_parameter(fn: Callable, param_name: str) -> bool:
```

Comment:
There is already [a function](https://github.com/appliedAI-Initiative/pyDVL/blob/4f65403278b4f77956bf7c0ef50250f5a5b4dcf2/src/pydvl/utils/types.py#L47) that does something similar.

Reply:
Merged both functions.